doc/source/conf.py

```diff
@@ -78,26 +78,6 @@
     "rllib/package_ref/utils/*",
 ]
 
-# Prune deep toc-trees on demand for smaller html and faster builds.
-# This only effects the navigation bar, not the content.
-if os.getenv("FAST", False):
```

Comment:
should we keep this `FAST` feature, but just do it on the CI (the `make html` line)?

Reply:
No, in fact we should not have been doing `FAST` on the CI jobs up to this point anyway. In principle CI should try to find issues with the docs before they get deployed. If we're trimming a bunch of toctrees that _don't_ get trimmed when we deploy, we're opening ourselves up to the possibility of missing something we should have caught on CI but only see in deployment.

The one exception to this that occurred to me is that of the linkcheck CI job. We actually might not have to build _any_ toctree for that job, which could speed it up significantly. But I think for the moment that this is best until we think about getting clever about linkcheck.